qiskit/transpiler/passes/scheduling/base_scheduler.py

```diff
@@ -261,10 +261,21 @@ def _get_node_duration(
     ) -> int:
         """A helper method to get duration from node or calibration."""
         indices = [dag.find_bit(qarg).index for qarg in node.qargs]
+        is_mid_circuit = any(
+            dag.descendants(successor) != set() for successor in dag.successors(node)
+        )
 
         if dag.has_calibration_for(node):
             # If node has calibration, this value should be the highest priority
             cal_key = tuple(indices), tuple(float(p) for p in node.op.params)
+            if isinstance(node.op, Reset):
+                raise UserWarning(
+                    "Reset durations reported from backend are currently untrustworthy. Do not rely on on scheduling results."
```

Comment:
Is this true for all backends, or just IBM backends? If it's just IBM backends via `qiskit-ibm-provider` or `qiskit-ibm-runtime` we might to reword this a bit more carefully because it might not be a constraint on other backends.